kolibri/plugins/facility/assets/src/views/FacilityConfigPage/index.vue

```diff
@@ -213,6 +216,13 @@
   import ChangePinModal from './ChangePinModal';
   import RemovePinModal from './RemovePinModal';
 
+  /**
+   * Use the crossComponentTranslator to aid concatenation
+   * of strings missed before string freeze. This only a workaround
+   */
+  const DeviceSettingsPageStrings = crossComponentTranslator(DeviceSettingsPage);
```

Comment:
I don't recall the exact details, but I think that defining `crossComponentTranslator`s in module scope can cause issues. I think this happens because something gets set prior to the initialization of the i18n machinery in the frontend, and so the strings will often be returned in the default language (English).

Best to set these up in the `created` hook of the component, and set them on the component `this` instead, e.g.:

```
created() {
  this.DeviceSettingsPageStrings = crossComponentTranslator(DeviceSettingsPage);
  this.PinAuthenticationModalStrings = crossComponentTranslator(PinAuthenticationModal);
},
```

And then reference them accordingly within the component.

Like I said, I cannot remember the precise cause of this, but might explain some of the issues reported by reviewers!

Reply:
Using the `mounted` hook seems to break the button. I have used the `created` hook instead and hopefully this resolves the issue.